ssas/service/admin/api.go

```diff
@@ -3,10 +3,39 @@ package admin
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/CMSgov/bcda-app/ssas"
 	"net/http"
+
+	"github.com/CMSgov/bcda-app/ssas"
 )
 
+func createGroup(w http.ResponseWriter, r *http.Request) {
+	gd := ssas.GroupData{}
+	err := json.NewDecoder(r.Body).Decode(&gd)
+	if err != nil {
+		http.Error(w, "Failed to create group due to invalid request body", http.StatusBadRequest)
+		return
+	}
+
+	g, err := ssas.CreateGroup(gd)
```

Comment:
I think we might want to log this [like Emily did in her recent PR](https://github.com/CMSgov/bcda-app/blob/ff40b28036af6e692384298fe2f1d08f207e16f8/ssas/service/admin/api.go#L25)